apps/src/studio/studio.js

```diff
@@ -1142,11 +1144,13 @@ Studio.clearEventHandlersKillTickLoop = function() {
     });
   }
   Studio.eventHandlers = [];
-  if (Studio.intervalId) {
-    window.clearInterval(Studio.intervalId);
-  }
+  if (Studio.perExecutionTimeouts) {
+    Studio.perExecutionTimeouts.forEach(function (timeout) {
+      clearInterval(timeout);
+    });
+   }
```

Comment:
Nit. I think it would be cleaner to make sure to give perExecutionTimeouts an initial value (of []) in our init code so that we can depend on it always existing, and then here we don't need to do an extra existence check.